<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>221</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>PWM</string>
    </property>
    <widget class="QLCDNumber" name="lcd_pwm">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>131</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
    <widget class="QDial" name="dial_pwm">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>131</width>
       <height>151</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="sliderPosition">
      <number>0</number>
     </property>
     <property name="tracking">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="notchTarget">
      <double>10.000000000000000</double>
     </property>
     <property name="notchesVisible">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>491</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string>ADC</string>
    </property>
    <widget class="PlotWidget" name="graphWidget" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>381</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
    <widget class="QProgressBar" name="progress_adc">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>20</y>
       <width>51</width>
       <height>211</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLCDNumber" name="lcd_adc">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="smallDecimalPoint">
      <bool>false</bool>
     </property>
     <property name="digitCount">
      <number>3</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>611</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Output</string>
    </property>
    <widget class="QPushButton" name="button_relay3">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Relay3</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_relay1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Relay1</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_relay2">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>191</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string>Relay2</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
